### PR TITLE
fix: last-seen overflow

### LINKED ISSUE #3038: User last seen overflow
image

### PR DESCRIPTION
**Pull request by kunatastic**
closes #3038

image

### CODE REVIEW

File: src/Components/Users/ManageUsers.tsx [hunk 1]
```diff
@@ -324,25 +324,23 @@ export default function ManageUsers() {
...
+                <div className="flex-shrink-0 text-sm mt-1 mb-2 text-gray-600 min-width-50">
+                  Last Online:{" "}
+                  <span
+                    aria-label="Online"
+                    className={
+                      "shrink-0 inline-block h-2 w-2 rounded-full " +
+                      (moment().subtract(5, "minutes").isBefore(user.last_login)
+                        ? "bg-primary-400"
+                        : "bg-gray-300")
+                    }
```
Comment (developedBySJ) [inline src/Components/Users/ManageUsers.tsx]: Use `clsx`
```suggestion
                    className={clsx(
                      "shrink-0 inline-block h-2 w-2 rounded-full" ,
                      moment().subtract(5, "minutes").isBefore(user.last_login)
                        ? "bg-primary-400"
                        : "bg-gray-300"
                        )
                    }
```

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Comment by nihal467** [comment]
@kunatastic @developedBySJ once its completed change the status to need testing

File: src/Components/Users/ManageUsers.tsx (hunk 1 above)
Comment (kunatastic) [reply src/Components/Users/ManageUsers.tsx]: done

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Review by cp-Coder — APPROVED** [review]

**Comment by khavinshankar** [comment]
@developedBySJ can you please rereview this?

**Review by Ashesh3 — APPROVED** [review]

**Comment by gigincg** [comment]
@nihal467 This looks super weird in Desktop View. Why is this approved.

image

@kunatastic Use flex-wrap so that it goes over only when there isn't enough space

**Pull request closed without merging**